src/taco-cli/cli/kit.ts

```diff
@@ -779,22 +779,20 @@ class Kit extends commands.TacoCommandBase {
 
     /* tslint:disable:member-ordering */
     // tslint doesn't handle this case and considers subcommands as member function
-    public subcommands: commands.ICommand[] = [
+    public subcommands: commands.ISubCommand<Kit>[] = [
         {
             // List kits
             name: "list",
-            run: Kit.list,
-            canHandleArgs(commandData: commands.ICommandData): boolean {
-                return !commandData.remain[0] || commandData.remain[0] && commandData.remain[0].toLowerCase() === "list";
+            run: (command, commandData) => command.list(commandData),
+            canHandleArgs: (command, commandData) => {
```

Comment:
I was a little confused for a moment about what `this` would be since the fat arrow here isn't in a context where `this` already exists. It ends up being fine, but is there a reason you preferred it over the other style?